lib/audit/catalog_to_moab.rb

```diff
@@ -5,7 +5,7 @@
 class CatalogToMoab
 
   # allows for sharding/parallelization by storage_dir
-  def self.check_version_on_dir_of_batch(last_checked_b4_date, storage_dir, limit)
+  def self.check_version_on_dir(last_checked_b4_date, storage_dir, limit)
     # TODO: ensure last_checked_version_b4_date is in the right format for query - see #485
     pcs = PreservedCopy.least_recent_version_audit(last_checked_b4_date, storage_dir).limit(limit)
```

Comment:
doesn't there need to be some sort of clause somewhere to set limit to Settings.c2m_sql_limit if it is nil?  Like maybe in the args `limit = Settings.c2m_sql_limit` ?  

I think we want this method to be directly callable.

Reply:
i like the default limit idea.  would be happy to do it as a follow on ticket, but seems worth doing.